<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSMainWindow</class>
 <widget class="QMainWindow" name="MSMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MSMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="lTabInfo_Widget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>1081</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="History_View">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>241</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchFor_TextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SearchFor_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search for : </string>
    </property>
   </widget>
   <widget class="QLabel" name="History_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>History :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1359</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_MovieSearch"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_MovieSearch">
   <property name="text">
    <string>About MovieSearch</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
